.github/config.yml

```diff
@@ -0,0 +1,37 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  Thanks for opening your first issue! This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.
+  Please follow the issue template to help us help you 👍🎉😄
+  If you have screenshots [or a gif](https://publiclab.org/gifs/) to share demonstrating the issue, that's really helpful! 📸
+  Do join our [Gitter channel](https://gitter.im/publiclab/publiclab) for some brainstorming discussions. 
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: |
+ Thanks for opening this pull request!  This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.
+ There may be some errors, **but don't worry!** We'll work through them with you! 👍🎉😄
+ It would be great if you can tell us your Twitter handle so we can thank you properly?
+ 
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: |
+ Congrats on merging your first pull request! 🙌🎉⚡️
+ Your code will likely be published to PublicLab.org in the next few days, but first it will be published to https://stable.publiclab.org/ (it will take some minutes for this to load, and until then you may see logs from the build process). Please test out your work on this testing server and report back with a comment that all has gone well!
```

Comment:
hey, you missed this line again! This is specific for plots2 project, as code from here is not published on stable.publiclab.org and publiclab.org...does that make sense?

Reply:
Hi @cesswairimu the correction has been effected. Sorry for the errors

Reply:
The unnecessary lines have been removed